job-service/src/main/java/com/hpe/caf/services/job/api/JobsPut.java

```diff
@@ -75,6 +63,7 @@
      */
     public static String createOrUpdateJob(final String partitionId, String jobId, NewJob job) throws Exception {
         try {
+            final boolean hasDependencies;
```

Comment:
Surprising to see this from a non-C developer (because in C variables have to be declared at the start of functions).  In modern languages they should be declared as close to their initial usage as is sensible (i.e. just above the `jobCreated` declaration in this case).

Reply:
Moved down.